### ISSUE TITLE
Evaluate on file open

### ISSUE THREAD

**Issue by stefan-toubia** [BetterThanTomorrow/calva#483]
The help text for the `calva.evaluateOnSave` states this works for when a file is opened as well, but it looks like this is not implemented.  Did it used to be, are there some undesirable side effects of doing this?  I'd imagine it could be a little much if you're just flipping through files, but I'd think the trade-off is worth it since a file needs to be evaluated in order to get documentation and symbol navigation.

**Comment by bpringe** [comment]
I don't think we want to eval files upon opening, because the user may not know what's in a file before reading it to know if they want to eval it. In my opinion it should be left up to the user.

@pez, if you agree, then we just need to change the setting description.
